samples/Ballerina/hello-world-service.bal

```diff
@@ -0,0 +1,16 @@
+import ballerina.lang.messages;
```

Comment:
Where do these samples come from?

Reply:
Hi @pchaigno,

These samples are taken from [samples](https://github.com/ballerinalang/ballerina/tree/master/samples/ballerina-by-example/examples) in the Ballerina repo which has Apache 2 license. If these samples are not enough, I can write/add more samples as well.

Reply:
No need for the extra samples @Shan1024, you've done enough. =) Ballerina's codebase has [some pretty detailed licensing info](https://github.com/ballerinalang/ballerina/blob/master/LICENSE.txt) to back that up, so licensing is nothing we need to worry about here.